news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+
+[![](https://assets.ppy.sh/contests/186/banners/Scissors_Mori.jpg)](https://assets.ppy.sh/contests/186/winners/Scissors_Mori.png)
+
+**1750** ghostly spirits asked which witch is which, and the answer was [**Scissors_Mori**](https://osu.ppy.sh/users/31302044)'s take on our beloved pink-fringed pippi! Witches really are vogue this year, but with costumes *this* cute, it's sort of inevitable, isn't it?
+
+[![](https://assets.ppy.sh/contests/186/banners/1_zero.jpg)](https://assets.ppy.sh/contests/186/winners/1_zero.png)
+
+Behold! **1547** voter-bats circled the spooky manor depicted by [**1 zero**](https://osu.ppy.sh/users/7440794), where a witch pippi and an animal-girl jiangshi Mocha prowl looking for delicious candy treats! A terrifying ghostly feline stands guard with his broom, ready to sweep spirits and scores both.
+
+[![](https://assets.ppy.sh/contests/186/banners/Amplui.jpg)](https://assets.ppy.sh/contests/186/winners/Amplui.png)
+
+[**Amplui**](https://osu.ppy.sh/users/24820805) wowed **1545** voters with their vivaciously vibrant and nefariously neon demon pippi! Eating an entire box of highlighter pens has never seemed to attractive until now! We're all but certain this one is really going to draw eyes in-game — a real eye-catcher for sure!
+
+[![](https://assets.ppy.sh/contests/186/banners/SuperSSun.jpg)](https://assets.ppy.sh/contests/186/winners/SuperSSun.png)
+
+**1437** voters supported [**SuperSSun**](https://osu.ppy.sh/users/6972778) stupendous sketch, showcasing several spooky mascots in all of their artbook glory! It's wonderful to see artists continually playing with new styles and approaches like this, and we're looking forward to seeing what both this artist and many more to come make with their incredible creativity!
+
+[![](https://assets.ppy.sh/contests/186/banners/MinMin_9s.jpg)](https://assets.ppy.sh/contests/186/winners/MinMin_9s.png)
+
+**1375** voters were wowed by [**MinMin_9s**](https://osu.ppy.sh/users/7140649)'s beautifully simplistic take on a cutesy witch pippi accompanied by her bedsheet-ghost retinue of Mocha and Aiko. The pigtails sticking out of the sheet is simply genius. Why didn't we think of that?!
+
+[![](https://assets.ppy.sh/contests/186/banners/RedcXca.jpg)](https://assets.ppy.sh/contests/186/winners/RedcXca.png)
+
+Fanart contest veteran [**RedcXca**](https://osu.ppy.sh/users/14056601) returns as a finalist once again, and can now proudly claim 'I maid this'. Because they put Yuzu in a maid dress, and we're frankly astonished by how well it works. Alisa in a bunny suit is also present, as well as a guest cameo from the ever-popular Doraemon for reasons we're not even going to pretend to understand, plus several trollface skull decorations. There's more hidden in this piece to boot, but I've got too many *tasks* to do to list them. 
```

Comment:
```suggestion
Fanart contest veteran [**RedcXca**](https://osu.ppy.sh/users/14056601) returns as a finalist once again, and can now proudly claim "I maid this", because they put Yuzu in a maid dress, and we're frankly astonished by how well it works. Alisa in a bunny suit is also present, as well as a guest cameo from the ever-popular Doraemon for reasons we're not even going to pretend to understand, plus several trollface skull decorations. There's more hidden in this piece to boot, but I've got too many *tasks* to do to list them. 
```